rali/rali_pybind/amd/rali/ops.py

```diff
@@ -789,6 +901,18 @@ def rali_c_func_call(self, handle):
         output_arr = b.CreateIntRand(self._values, self._frequencies)
         return output_arr
 
+class Uniform():
+    def __init__(self, range = [-1,1], device = None):
```

Comment:
Codacy found an issue: [Redefining built-in 'range'](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=5763058)